canal/dump.go

```diff
@@ -163,7 +164,7 @@ func (c *Canal) dump() error {
 		if err != nil {
 			return errors.Trace(err)
 		}
-		c.cfg.Logger.Infof("skip master data, get current binlog position %v", pos)
+		c.cfg.Logger.Info("skip master data, get current binlog position", slog.Any("pos", pos))
```

Comment:
```suggestion
		c.cfg.Logger.Info("skip master data, get current binlog position", slog.Any("position", pos))
```

We should aim to have meaningful and easy to understand names that are consistent. Might be better to use "position" instead of "pos" for the key of the value here. Feel free to ignore if you don't agree.